fluent-syntax/src/ftlstream.js

```diff
@@ -109,14 +109,50 @@ export class FTLParserStream extends ParserStream {
     return ((cc >= 48 && cc <= 57) || cc === 45); // 0-9
   }
 
-  isPeekNextLineComment() {
+  // -1 - any
+  //  0 - comment
+  //  1 - group comment
+  //  2 - resource comment
+  isPeekNextLineComment(level = -1, oldStyle) {
```

Comment:
I know this is a temporary, but how about we rename `oldStyle` to `zeroFourSyntax` to make it explicit?